portal-impl/src/com/liferay/portal/service.xml

```diff
@@ -287,21 +287,35 @@
 		<reference entity="User" package-path="com.liferay.portal" />
 		<reference entity="Website" package-path="com.liferay.portal" />
 	</entity>
-	<entity local-service="false" name="Country" remote-service="true">
+	<entity local-service="false" name="Country" remote-service="true" uuid="true">
 
 		<!-- PK fields -->
 
 		<column name="countryId" primary="true" type="long" />
 
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
 		<!-- Other fields -->
 
-		<column name="name" type="String" />
+		<column localized="true" name="name" type="String" />
+		<column name="billingAllowed" type="boolean" />
+		<column name="shippingAllowed" type="boolean" />
 		<column name="a2" type="String" />
 		<column name="a3" type="String" />
 		<column name="number" type="String" />
+		<column name="subjectToVAT" type="boolean" />
 		<column name="idd" type="String" />
 		<column name="zipRequired" type="boolean" />
+		<column name="position" type="double" />
 		<column name="active" type="boolean" />
+		<column name="channelFilterEnabled" type="boolean" />
```

Comment:
We should probably rename this to `groupFilterEnabled`.  Based on Marco's comment in [slack conversation](https://liferay.slack.com/archives/CS52K673L/p1603390103006800?thread_ts=1603388687.002600&cid=CS52K673L), this will be extended to include sites and other groups later.

Reply:
Updated, thanks.